src/main/java/org/jabref/model/entry/LinkedFile.java

```diff
@@ -1,23 +1,31 @@
 package org.jabref.model.entry;
 
 import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
 import java.util.Objects;
+import java.util.Optional;
 
-public class ParsedFileField {
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.metadata.FileDirectoryPreferences;
+import org.jabref.model.util.FileHelper;
 
-    private static final ParsedFileField NULL_OBJECT = new ParsedFileField("", "", "");
+public class LinkedFile {
```

Comment:
Could you add a JavaDoc stating the intension and usage of this class? Also, why no TypedBibEntry is required anymore?